reana_server/status.py

```diff
@@ -236,7 +236,7 @@ def stuck_workflows(self):
         inactivity_threshold = datetime.now() - timedelta(hours=12)
         number = (
             Session.query(Workflow)
-            .filter(Workflow.status == RunStatus.running)
+            .filter(Workflow.status.in_([RunStatus.running, RunStatus.pending]))
```

Comment:
What do you think of moving this status check to its own new entry, e.g. `unschedulable`, for more granularity? In general, it would be more alarming to see "stuck in pending"/unschedulable workflows that "stuck in running", the latter can be a false positive (just a very long workflow), whereas the former indicates a problem with the system not being able to schedule pods. Also, a quick question, what would happen with a `pending` workflow that won't have the `Workflow.run_started_at` set?

That new entry could go instead of `pending` and we could say something like e.g. if a workflow has been in `pending` for more than 20 minutes, it is definitely `unschedulable`.